docs/json_rpc_api.md

```diff
@@ -675,6 +676,16 @@ The following is an overview of the RPC methods which are implemented by dcrd, b
 |Returns|`[ (json array of strings)` <br/>&nbsp;&nbsp; `"blockhash", ... hash of the generated block` <br/>`]` |
 [Return to Overview](#MethodOverview)<br />
 
+<a name="getstakeversions"/>
+
+|   |   |
+|---|---|
+|Method|getstakeversions|
+|Parameters|1. hash (*chainhash.Hash, required) - The hash of the block being queried for stake versions |
```

Comment:
As you can see you're missing the optional count and the returned result isn't specified.